package/setup.py

```diff
@@ -295,6 +296,15 @@ def extensions(config):
     else:
         encore_compile_args.append('-O3')
 
+    # allow using custom c/c++ flags and architecture specific instructions.
+    # This allows people to build optimized versions of MDAnalysis.
+    # Do here so not included in encore
+    extra_cflags = config.get('extra_cflags', default=False)
+    if extra_cflags:
+        flags = extra_cflags.split()
+        for flag in flags:
+            extra_compile_args.append(flag)
```

Comment:
couldn't this just be done in one line using list.extend()?

i.e.

```python
extra_compile_args.extend(flags)
```

Reply:
Yep for sure